apps/rush-lib/src/logic/StandardScriptUpdater.ts

```diff
@@ -12,7 +12,7 @@ import { RushConfiguration } from '../api/RushConfiguration';
  * This is used by the "rush install" and "rush update" commands.
  */
 export class StandardScriptUpdater {
-  private static readonly _scriptNames: string[] = [ 'install-run.js', 'install-run-rush.js' ];
+  private static readonly _scriptNames: string[] = [ 'install-run.js', 'install-run-rush.js', 'install-run-common.js' ];
```

Comment:
>install-run-common.js [](start = 95, length = 21)

instead of creating a library script, maybe we could put all the code in one script, and then have the other one simply communicate which behavior (e.g. via a global variable, or via the argv[0], or somesuch)